documentation/book/ref-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
```

Comment:
As mentioned above, I don't think this constitutes a standalone reference module. Suggest incorporating it into the assembly.